examples/classic/kubernetes-private.classic.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {

```

Comment:
Instead of adding another config for specifying this the absence of dnsPrefix on master pool can be used as a choice for no public ip. Otherwise if someone specifies  "noPublicIP": true and set dnsPrefix on master what would be the expected architecture?